gsa/src/web/entities/usePagination.js

```diff
@@ -61,3 +62,12 @@ export const usePagination = ({simpleFilter, filter, pageInfo, refetch}) => {
 
   return [getFirst, getLast, getNext, getPrevious];
 };
+
+usePagination.propTypes = {
+  filter: PropTypes.filter.required,
+  pageInfo: PropTypes.object.required,
+  refetch: PropTypes.func.isRequired,
+  simpleFilter: PropTypes.filter.required,
+};
```

Comment:
I don't think propTypes are used for hooks. But i could be wrong.

Reply:
Hm not quite sure... I can just remove them.